.changeset/dirty-rings-cry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@firebase/functions-exp': patch
```

Comment:
Please remove `@firebase/functions-exp`. Exp packages should not be included in changeset.